common/config.go

```diff
@@ -83,8 +83,8 @@ func StagenetConfig() *Config {
 				Port: 38081,
 			},
 		},
-		SwapFactoryAddress:       ethcommon.HexToAddress("0x3d561C6f938aDBc45239772cc6A39e1Db7192154"),
-		ForwarderContractAddress: ethcommon.HexToAddress("0x4a707181842Ef084daFC90DeF367a1825eCcBCab"),
+		SwapFactoryAddress:       ethcommon.HexToAddress("0xE5656d3549478BdE248e5D7aE22ff8DCC5Cd11a3"),
+		ForwarderContractAddress: ethcommon.HexToAddress("0x171f294A0Ca7085Ce5F73DE1BE28b9721e007B94"),
```

Comment:
Side note, not necessarily for this pull request: I'm thinking we can just get rid of the `ForwarderContractAddress` field just load it from the SwapFactory contract. Then we'd have one less thing to keep updated in the code.

Reply:
will open an issue for this!